src/CoreArbiterServer.cc

```diff
@@ -1017,11 +1016,28 @@ CoreArbiterServer::distributeCores()
         size_t numCoresToMakeManaged =
             numAssignedCores - managedCores.size();
         LOG(NOTICE, "Making %lu cores managed", numCoresToMakeManaged);
-        managedCores.insert(managedCores.end(),
-                              unmanagedCores.end() - numCoresToMakeManaged,
-                              unmanagedCores.end());
-        unmanagedCores.erase(unmanagedCores.end() - numCoresToMakeManaged,
-                              unmanagedCores.end());
+        // Ensure hypertwin of managed core not managed unless unavoidable.
```

Comment:
I don't believe this is the behavior we want. What we want is for the hypertwin to be managed but idle. If you leave it unmanaged, the kernel will run other stuff there, which will interfere with the thread on the other hypertwin. Let's discuss.